gulpfile.babel.js

```diff
@@ -57,10 +54,10 @@ const vendorStyles = [
 ];
 
 gulp.task('clean', () => {
-  del.sync([`${publishDir}`, `${buildDir}`, `${clientDir}/data/*.json`]);
+  return del([`${publishDir}`, `${buildDir}`, `${clientDir}/data/*.json`]);
```

Comment:
Gulp now supports promises, so we can return the promise from `del` in this scenario instead.